activestorage/lib/active_storage/service/disk_service.rb

```diff
@@ -123,6 +123,32 @@ def url_for_direct_upload(key, expires_in:, content_type:, content_length:, chec
       end
     end
 
+    def public_url(key, filename:, content_type:, disposition:)
```

Comment:
This is also still a signed URL without expiry. I assume we may want to attach metadata and/or encode the key to the asset? If not I can probably just make another controller action in `ActiveStorage::DiskController` that uses the unencoded key as a param.